packages/runtime/src/client/client-impl.ts

```
@@ -110,18 +101,13 @@ export class ClientImpl<Schema extends SchemaDef> {
-                    adapter,
-                    connectionProvider,
-                    []
-                ),
+                executor: new DefaultQueryExecutor(compiler, adapter, connectionProvider, []),
             });
         }
 
         this.kysely = new Kysely(this.kyselyProps);
 
-        return createClientProxy(this as unknown as ClientContract<Schema>);
+        return createClientProxy(this);
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)